Analysis/R/prepare_stan_input.R

```diff
@@ -434,6 +448,27 @@ prepare_stan_input <- function(
       obs_changer <- setNames(seq_len(length(non_na_obs_resized)),non_na_obs_resized)
       stan_data$map_obs_loctime_obs <- as.array(obs_changer[as.character(ind_mapping_resized$map_obs_loctime_obs)])
       stan_data$map_obs_loctime_loc <- as.array(ind_mapping_resized$map_obs_loctime_loc)
+      
```

Comment:
this is repeated because we're dropping data for location-time slices with 0 population and re-compute everything into the stan format in the event that indexes are dropped